bower.json

```diff
@@ -25,5 +25,8 @@
     "ember-cli-shims": "stefanpenner/ember-cli-shims#0.0.3",
     "jquery": "~2.1.1",
     "ember-data": "~1.0.0-beta.10"
+  },
+  "resolutions": {
+    "ember-data": "canary"
```

Comment:
Ah, sorry, I must not have enough context in ember-mocha, didn't realize we were testing against ember canary here.  I'm just after the ember-test-helpers upgrade trying to get to latest release version of ember.